zuul.d/jobs.yaml

```diff
@@ -421,6 +421,34 @@
     vars:
       ansible_test_python: 3.6
 
+- job:
+    name: ansible-test-integration-base
```

Comment:
we should parent to ansible-test-base, to remove a lot of the duplicate code in our playbooks